adam-core/src/main/scala/org/bdgenomics/adam/converters/SupportedHeaderLines.scala

```diff
@@ -63,10 +108,72 @@ private[adam] object SupportedHeaderLines {
     transcriptEffects
   )
 
+  lazy val genotype = VCFStandardHeaderLines.getFormatLine(
+    VCFConstants.GENOTYPE_KEY)
+  lazy val genotypeQuality = VCFStandardHeaderLines.getFormatLine(
+    VCFConstants.GENOTYPE_QUALITY_KEY)
+  lazy val allelicDepth = new VCFFormatHeaderLine("AD",
+    VCFHeaderLineCount.R,
+    VCFHeaderLineType.Integer,
+    "Allelic depths for the ref and alt alleles in the order listed")
+  lazy val formatReadDepth = VCFStandardHeaderLines.getFormatLine(
+    VCFConstants.DEPTH_KEY)
+  lazy val minReadDepth = new VCFFormatHeaderLine("MIN_DP",
+    1,
+    VCFHeaderLineType.Integer,
+    "Minimum DP observed within the gVCF block")
+  lazy val phredLikelihoods = VCFStandardHeaderLines.getFormatLine(
+    VCFConstants.GENOTYPE_PL_KEY)
+  lazy val strandBiasComponents = new VCFFormatHeaderLine("SB",
+    4,
+    VCFHeaderLineType.Integer,
+    "Per-sample component statistics which comprise the Fisher's Exact Test to detect strand bias.")
+  lazy val phaseSetId = new VCFFormatHeaderLine(VCFConstants.PHASE_SET_KEY,
+    1,
+    VCFHeaderLineType.Integer,
+    "Phase set ID")
+
+  // note: this is not spec compliant!
+  // however, this is due to a bug in htsjdk --> https://github.com/samtools/htsjdk/issues/751
+  // the standard header lines get populated no matter what the header lines are on the
+  // file. not clear how to disable this...
```

Comment:
Yeah, I wasted an hour on this last night. The relevant code path is https://github.com/samtools/htsjdk/blob/master/src/main/java/htsjdk/variant/vcf/AbstractVCFCodec.java#L224. If your header line disagrees with the standard header line, HTSJDK clobbers it by default. We can disable that, but I don't think Hadoop-BAM supports disabling the header line clobbering.